<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListWidget</class>
 <widget class="QWidget" name="ListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ListWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextBrowser" name="textDisplay"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="valueCheck">
         <property name="text">
          <string>CheckBox</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="valueSpin"/>
       </item>
       <item>
        <widget class="QCheckBox" name="textCheck">
         <property name="text">
          <string>CheckBox</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textLine"/>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>Clear All</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="quitButton">
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>valueCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>valueSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLine</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>textDisplay</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>ListWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
